### ISSUE TITLE
Reader: poll for changes in the comment list

### ISSUE THREAD

**Issue by designsimply** [Automattic/wp-calypso#792]
We should poll for changes in the comment list to show new comments.

Steps to reproduce:
1. Go to https://wordpress.com/ and click on one of your blog posts to open it in a single-post view
2. In a separate browser window, add and approve a new comment on the post
3. Watch to see if the comment shows up

<strong>Result:</strong> new comments don't show up automatically when you are viewing a single post in the Reader and a new comment comes in.

**Comment by alisterscott** [comment]
Confirmed this behaviour has not changed - comments do not dynamically update in the Reader

**Comment by bluefuton** [comment]
I started some of the plumbing for this in https://github.com/Automattic/wp-calypso/pull/26318 but it's not exposed on the UI yet. @westi suggested we use Pinghub rather than polling in the background.

**Comment by bluefuton** [comment]
Just noting that @hewsut's https://github.com/Automattic/wp-calypso/pull/40396 adds real-time comments to Reader full post.

**Comment by bluefuton** [comment]
This should have been fixed by #40396 but that will need to be confirmed (cc @worldomonation)

**Comment by worldomonation** [comment]
I checked this behavior across multiple browsers and both Atomic and Simple.

I was not able to observe the real time comment fetching in a full page post.

Steps I took:

1. log in as User A.
2. open the post under test.
3. as user A, in a separate tab open the Comments management section by going to My Sites > Site > Comments.
4. as an unlogged in user, enter some comment on the post under test.
5. as User A, refresh the tab used in Step 3.
6. as User A, approve the comment.
7. as User A, switch back to the the post under test.

The steps above were not able to demonstrate real-time comment loading when viewing a single post on both Atomic and Simple sites.

On Browser: Firefox Nightly 87.0a1 with Developer Tools > Network open, I was not able to observe any network requests being made.

Video demonstrating on an Atomic site:

https://user-images.githubusercontent.com/6549265/108105131-14dac980-7041-11eb-82bd-c9fb4742ec98.mov
